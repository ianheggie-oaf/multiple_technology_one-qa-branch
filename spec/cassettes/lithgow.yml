---
http_interactions:
- request:
    method: get
    uri: https://lithgowcc-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationSearchResults.aspx?Field=S&Period=L14&f=$P1.ETR.SEARCH.SL14&r=P1.WEBGUEST
    body:
      encoding: US-ASCII
      string: ''
    headers:
      Accept-Encoding:
      - gzip,deflate,identity
      Accept:
      - "*/*"
      User-Agent:
      - Mechanize/2.7.7 Ruby/2.5.8p224 (http://github.com/sparklemotion/mechanize/)
      Accept-Charset:
      - ISO-8859-1,utf-8;q=0.7,*;q=0.7
      Accept-Language:
      - en-us,en;q=0.5
      Host:
      - lithgowcc-web.t1cloud.com
      Connection:
      - keep-alive
      Keep-Alive:
      - '300'
  response:
    status:
      code: 200
      message: OK
    headers:
      Content-Type:
      - text/html; charset=utf-8
      Transfer-Encoding:
      - chunked
      Connection:
      - keep-alive
      Date:
      - Wed, 19 Feb 2025 03:31:57 GMT
      Strict-Transport-Security:
      - max-age=31536000; includeSubDomains; preload
      X-Content-Type-Options:
      - nosniff
      Cache-Control:
      - no-cache, no-store
      Pragma:
      - no-cache
      Expires:
      - "-1"
      Server:
      - Microsoft-IIS/10.0
      Set-Cookie:
      - ASP.NET_SessionId=jq2li255tke1zf55y2iql045; path=/;SameSite=None;Secure; secure;
        HttpOnly
      Content-Encoding:
      - gzip
      Vary:
      - accept-encoding
      X-Cache:
      - Miss from cloudfront
      Via:
      - 1.1 cf05b482095d42dc6afa733c1fd95900.cloudfront.net (CloudFront)
      X-Amz-Cf-Pop:
      - MEL52-P1
      X-Amz-Cf-Id:
      - tWZYgAvHWOBcTYz-U0Rc1MaCPna8_6w7ynat99zyPgM_K9_kyS5w3Q==
    body:
      encoding: ASCII-8BIT
      string: !binary |-
        H4sIAAAAAAAE/807aXPayLafM1XzH/rykklS8wwIbzgB39LCaoFBLDLcuuUSUiPJaBupZRm/uv/9nW5JBDB24rmZSZw0Um+nz96nT8Ovv/z6S+0f0rU4ng0ayCKugwYTQe6IqHBUKqnHYqkkjSV00x73ZMQVy2hEQlsnpVKjX0AFi5DgU6mUJEkxOS76oVkaK6UHCoWj07LXo4jNKRrEKFyyBekI9OA6XlQ/AIO7uLhIp9LhNQtrxmWN2MTBMPuNbBPL9BMk2mSNRD/2dNtBR2gUL1ybEGwgWYsI4k6QpK0jIK6Uzqw5trdCVoiX9UKxWIL/fBDcji3s4qgk4aUWOyR/FvUoKiCyDnC9QPADKbF6iJ16ISJrB0cWxqSASpe1kkWRg1UWvrEG7H79BcFfLdJDOyDbEO60ey1tLaAo1HMcRmxgVLr7I8bh+ui4eFbkiq7tFe+iAkBPZ1wiyjSAi2pLP3SRp7mAmBYFHiZNaCggFxPLN+qFwI8AL00ntu/VC3gcavoKyHRsXaNNI6yFuqXgCGiNiloUPPyzaWPHqI9+09zg8wCHNkCRuRNWXdbfVU4GXLExVoqjBq+I7eIo7wvr0KE2hNakMRoXkA1rA7gcH2BEzbDv6cP2gjhnhGUbBvYKGf63t9NOQx2N+XEjBbDTcK85Ma4XSkljICWTq964cdqTzMeetCr3k4tV0+y6UmWo8mZHoE/B7DTYk0/rvDsbzIVZgzcb9CnOzWxOUgWsTxrT1vTE4CddRVUCQxT6i4ritE0iLSocmamnq7HbJPPhBMY1H7WHk8GwckEWajOeN5rr+fGUzNXTsqAEw95kOpys+px+rNwvVOdxdqMEi8ppc95yymP1YjVXTrpDtX83v+k/jirTU6gPh5Ump1emj1O3GWmqYhkt537hmlhsKtZs5cTz1vSEH/J82zQmi8pFNG05yVw5GeyMXxnrxfE0mavKCnhwgMYqLwy50Vydr2Ftv9PoW7OK5bSTRlOAdfRKtOTNCZ5Izeak2ZWMm355UeECvdIsa+pFPHGnru5OH7XNvD7MVfx2MhF4nl8K7mQpVKarOd+RJOApb520p67zqKndaK5aIJ+OyJvVwZyvDppixPPKrNU2leuZyjnCMIj08pxbuH22Vtt8aACelqE+lCmNhjppNtbKsDc6bU6bnXhSngqTVb8jO/1ebzzEkiNYOpXPjeLoztxatKfOXBiG/LAh8hVlxZs65UeLyl56mIntRHDmraHIM72Bpwk4mZ0lz096k8oUYPWBP1QO08d03KZ/1pNm9z2pc9+749f9kVBerPmH61E16dnVNbRzdLwwpOOHWGwYa5DnBN8IQOMD4N4VhmUyVibN0bBs4kbaL+muU57fdIPZ5CHQj4fr8+GQ5+E/PyjBX5Xyl0//EpE+J83hg+zwUAxnZp8OZm7gzI4VYd5qJkZrutbXiTl14Vlx7hd35YeeeFKW7/i4Jybm8HgaGe3pej7m4vnNtAx8i2RRGBpqN9LUvjy/cSaLCnEWd1wM/I8EvgFLijo/7Z+CTjuL0am0qDxEc7XPZKWvT9ug+w7QcCevKP6K2ZuSxVRqxBMo08rUllcXtuZO74xGczW7ESitj/KK8eYL7SPAGca/OEcUVOAT6NRFPBg3EvluCDSdJIC/BLiWjZuuM1BPHeNGWc/UJOo0BW7mPgSzMuwUU+Veq0xj6OcWrQfjZtgDuvjWTQV4r3KPwvTiUbsJHLF5UcnXoIzvcl/j9WkbaF8Nj7sB5b2mNhc3ww6F3RH5lURfeCpLnvKRb/MJT/+EzQc/dF7NqxR+lwHhy5IwHFJwfwLOhM5rL2766kx94OZDi+jc3KL20zNPiV6+SIAXltG6WIO/W82HAbT176GfLFbOSnK5x0m7e6+3pmuj7VzPVM7hhw0BisjDn2Ay+OquzfAPvTs/6UlQREEYD2ctGMo37xqn8iO/lh87Se9uYvbGjbP+eEWLOWmUWwLjW7cxNGeMudLk4X7eMpyFOo2NBrXVSZctypNVM9sLBL4jSJWhytM9gfmkKi8MHSavTlM5Bby3/FW1o7X2xioP14u1ECzc+b3ucsAHsCdbCPS1YBluM1i0mvaiNYmpzUvDqspbDMaMT6YS2GPUk/jVsHIRg17EhqjcgU8dge/lFm3lUQF9mYvKen4j3MzVLsOp2+qXFxKfdFvGet4E214xG5nM1C4Ha63BnzpGUq1eJfwVPzRFXh02h/mcBsjFU5yFN1zNjqeP4K8fqU3oZQpjuALfnVBbzNa9M1pJ0hOVldGe3+Trj1vOo9GcB/MbnfloAXznvKUAP3lRrAyb7QNrLVrdaFKZMp+ptJpg272B1lIc9+yPO3I+9sZq2RZmd1b57vfrx1XXv+LrdRooQYRUMlhEAG/5y5vaN0UHrUa/ofDja+VJmLDVk8ULDb56Jp5VztM1X16gMW30x1Ne7kj8uHPdz6E/ac5AQyiidvgr6XcZl8Xh1fHpRftKSuTj6u+jsjCUpfK8VBKh/y54xPAU/5ipsmRe/c4FrTvJlOOQeKciL0dx9zzgTXnlz9QRzOvG18uqaF5Jvs/qflbXgg6rl+KY9stC2n/lxpjVzbQuN7N6Kauv0vpVK4Mfkt/p+CvinVB4VyPC4F1h/9TrC4BvfB8fi6bc9Oes/kc8YPVVWpe7af2qZFbVnogHkTK6eTxOyurjbKR3Hx9Oh4bSCNbXbsryWiZmGq3SUgNhs/DQ8TXD9szCJW3dKmnETBuenapHkeDHnhHR48CX8WxODp6O0TwPhwUEYSft2i+1fxwdIdk3bQ8dHT03hoLTHS2K6oUowLqtOY7trei6++Dy+i4+eWv+rEWB5qEMIlv8JVibWRpjmU6ccvkWPtnEW9012EsBabqOo2iF1/WCDOcUejKqF3jWiK7w+hOCVivEy3rhTsuPHZ9ubw1/4EdE0PTVh/cAtVx+C58M5Nsc9vv/ff/+Y+GSNdZK2r6wcgQ3z5epR7USpf8ZZudAnoGxrw75cPpksmx4cNbL5Ulb8/IU4EYLN2oCFPrPiaKmZedEi5AggvOtYxPL9BNdP0rwokg43fFjo6j7bmnMDRQJL7XYISUVL/ggiEp4EPoBDsm6NOBKbd/F6Ulr58j027Iui8XGQCm2r3uN4rSjpu4HBEJF7hp0Xl+7fw5FSmrNdk0UhZuzJCx+O7Z2zrMdVzNxVGq4AVkXTXu5vYhN+9qAHl0GLfzQwGG9UC7k2mrlXaVn5Ef14wCr9xwAw/RQ25ap6Q7WQnrehba9xbYlJ/oewR7hQ6wd4sv20LEWrQLNwwWUkoPqqACy7/uD0PbIodkUz6fU0FYKdw8r2pyXw5PyXvqkAFI06gUXe/FA87DzHA50Ai2vB9vGGojwa3ApbFpSz2Qb9UKqcznHbsMg7AGSsh0R6nvK3K2zcGhLK/TjIF8ElM2xdY3YvocUbMJgHH6TtbO1D+gDbd8vtdjZ5htDgOJFaXxTc+zLjaXicagdSHUcsjwNMh3Lp6mNRn9YuNymagQ6qVtUyWslWArillLsvKAHOfK1RQiZoZfF93IvhbSvMznfvz6TzaZud2sPeVa4la8KlzEWNm66dXa8Hyhi6ozBF+MIh/e2jqNi5pWLXpQUTf++qMVfXO8g9I2SaGmeiQewmyfg3VJdUPASh+ADzEnogBIsN86aznhXWQ44+KDOlw1/d7wM3x0bA66oNoRrFeLQd5WzJbS8q5xWTqB5x4NfpguifMWN7uyoKqXjv9xUHLrrMQT/uay/BTQAu6J83er02daNqJw0SB+C2r68NHn9hva1ta9jsqH777aZ0tP9g1rDdnnefg73vBSF5HCfznxu1oGRgHJqqFGUW6mCHY1gQ4bIM8o3sPqf378omsydfHH2im1ahC5Ht6JbEgW7S+ppBEy0aEUHsL2KQdjqyB1SNhMxbHO6wXGDp9zRvWc8tPKVZPRYZbnocDvfzFqWYL0noPrb2emxWrgcxQvXJsA+NLbsCKkYr7b1kZqEloV3fxIl+TUoyTsoyVpE/gqUxj3Gk2/kUu8Jl3oQWe1sdd+DTa/BSd7BibHpL8Hp/BV8ks+3+cRwOkeSto6+tz5l9y3fJj2ZO3mCVnbZ9b3xqlRfw61K9Qlelep34pfEEMnup17jEiTqEiRMcOjaHtwNfkefsIPTa3yCJINP2MKJadb38VM7OL3GKUhjMMAtnBifvpMF7iAlv8IrSNQrbCHFGPWXIPUKtyDJ4Bb2kfpefmGXVa/xCxLzC/t4fd0xaAdOTQf25F3EXuMYJJk6hn3EDniG9HS1H2Y0fR+Mt3D5m7eIgs+b+CKL9J7GUzTYoeVwz2taGSYsZIqiNnYC3jN4A7wIPXzSFQ6VFH7+mT73x+U0fGv7/rinULcxzXIkh5Bkw7IITo+ibKTqhys4Ler40BS6NpyxbZol2MwY+wHN1WgQ4dH+/fIUv3xEzeLSUJfQvGYaQN5a3AByVGOaw3yyf2zU1+JetdhhFA635sghdLg/bU0/a8Aob4uEjIW3ehw12Dl5K3+Qpg8yUyqgiKwdXC8YdhQ42vqT7UFeGR8tHF9ffU5sg1ifuHL53WfKWLpIRu52LuFlFJyFk64oaQTDqSGTM1sAgFKYBxCHWU0bO0Zfc/GAg1D6EIwlHSFrC5qzSvthV4dFslzAc0zJYU/pNzm+CXj5rFSulCrlyikiPuIu8lq+UIlSkXEm5cavv6T/4KThrdCTL+jsf4PHYjnxryRtQYa3WRo1xBEmR3oUFaEUaB4UTrSbAxvT4i0VyGTdCm3jNvAcMY6I79oRy5HRRmpgb3K8qdptv9M6LcxImb29BD7EAWSbKdBbnY2D9UZYp9k4alaQrmGLZatRuHl5Tlqgwtv4P1nAofq1swDKVCzaWxf459hZZjBTlwxyJseU7ByhQ89ttd/uZxnv1zPHds2MOSMcaKFG/HCT5c6w3+qIXplR1xxSL2wMPM2jH6UmXQ4ePlvYhrP2J/qaNp5SM3+SIfz1lzebjWGXQVTDd5gACghG8OZNjWgLB+diMEHtgCrsOGAlOmhAmsuHeqAZBqsfF7bT/K/noxkamdaNtUXsaOHUxsmG8pQ65sUyJui+42hBhD/lL+DegE6KeZijbbGkteInqcLSPgtFuh+As4Rp4Lq07LT+lauszBDf7iry2y+K/JZZytuDRMC118gPyVtFczsGvQDbVmKa2UiPVsS6rJEfiJ7sGybc73iEOnmK5qaBeeSfA0kJR3poB9QXURS3qj8HfoPQdrVwzS4TtpCjuLI2tNX4U2EsagSbfrjewo8inTf/fHg3/dDVCKTieMOArTSi2A6yu1GUtf0cHM7M3SM0EGJ4bloQa9qgmfqvEgnBE2ycmEcJdXacmPHlbuqvdFzMqVXST3BilMOiWC5XS5XTDGVAhBiXX0KrWgmqNSgjCxubCnjPiIQx2+CRCJfX9pLu4XgzQMGRbWCP2JqzaYOTgcUiqr+JwuOMwuNT1HGWfmigEQkxJmhwrYxlvi+h/khFlerJ+Q7tA8t27AD1tHCFFOxFlpYwEg6KEnZzHHpgaT9Mmsc70pT4cvliX5oc9yU0BkkyacYLw763I9v3UAU5PoEsZdYl4Xvs+IEL0kN8EOTBGeNBDcZI/DWx6O0tvFNQf59YU1I3Yj05Pjo+O0aq7Th+guDcgJHiawZSeVluKLzK91u5hHe1m4kWzueoiz3PXuIQ9XGi++4i1d+DgvZ+rM2ePJHyE5s9IGVwmjbdWJG/hFtGePsixJ9WyCmluZBFCBvxPUgYo7MTDo18I9EgZE0FPeYVvs+nMj6v7lrxCKJ2F4HTCkBXjciHq2jQ14Oi1Rzyg234dEe61CN/iw0PtG2J/uQuOSXxvxarRC02Ql0NJIqEdej9vCZ79kSo59/gmBVMNNuzPROpoOgbe/3JpZvSmku3UkFt7IU2JkTL99xvc8qKH0VI1NxgAQdRRvxParHn+8Llyt8gXHBHBqOK7ptfF2kc/pDoKaUtF2ZfC0EbHQ1x1fIJaoVYo1fSEXWZqA25ikRbo971pD9GMt8TOo3MHV+Ud4IqxVpHSAj9xHP8F8Ip78fustUdqY7iBQ1ty2XuiWgrTwIqBKM3ERWdZqcRMTpCjQfsBgRexn4YYi+Ct2MkQ8iFkO1B5rKyUYlnYHyJzXIILP+9mfb3RWApg3LVqJyfo77vg26wuEu4bjUaqfirJxc74ofDJsRphCCq9LFDQKMY9j+pdV/s6AHdj8++RQXGiY9QZGEj+oR8DyM7QtUTt4ISm1hIS6hXj/4XQfCMfPgIaT93VtkMgLQbgXhcc/KxRcajGsQtlG3kxfPWD/IVKadyheC4chUN7ACHEWo64CiYXgyuXzpusa1ijUbaw2LN6D2oEz/WMXDsq+/lSnagHJ1VyxzcPOyeKrjjJ05h5Du2gZoxdoBKOCSGjEAq0FGaBUdn1YNnrOcn/m22ntGci7aFPT9xNC+NvFtyo494uSM+Y+/g6b1k7SEZ2+AdJD+h4VobJ3ZkMRYclLHm0E0F2PSjztIct2/3cKbay4wckjJYPKOKCvbrlvojMyMZhblQOQ41bQ8uGBF/j70YI7kzbreu1dSJn+/t4bxHLN9bo65veUjUQpAWI/ugMH+wwe7muCArciA+O39qr9uSlL5yRFZ+qCBTAv+cIEe6D1uxjCPHxqiny9hPo9IvgqRvIFsNLozo3dHmqmnrYmn31unQFdzuiOdq9L4qh0/f00IXSmfUIKrKb380lnAUYkJ8L5LxktBbIDqUlp1fB0bsa50F5MFNdfaThbdgmgR75O2CeIMQkgp+DBff95oTw93RlwabfnmBXZJm4293xmu6nv+WalBAxCb0mn77t1TQqjtaFNULFEE+hJg3RTj9wRvDdPN1lJRE2rZdtknWHaw981uXfMoh3v83fbk0chj581D7wbbgslYKQG/Q1l8tyIVoezTvCJoFl3WlJ8OotJkIokjwQc6uAOTnU/UoeuEnOYxt+3NlbGrOk18Rohe/vYH2/g5L6TWtewDz6reCOMRlCuMgxaIfrEM4mIFx/AYXo+vPaIx1y/Md31yjawhLZdu14ZYF0W9OHIJ8qO0Lptu91FZr6fUd2vryxJ2W/4qQ2mepVPvHv0SJH/P/+vWXtx8MX49d7JGPxRBrxvrDMvaYVaMPH9H/2Uv04e2H9/+jMwME22yA/YRww1K0ow/vP2WK8/4jqtcRxMMYptDht7cjUbmW5cH1qDPuXPdv3n8swuQP78vvP34+NGC2PeA///n4uVT6979BY2ullBr2uoTDKHtZ+MaavVjEpT/p+X+tDdYHBUcAAA==
  recorded_at: Tue, 14 May 2019 14:00:00 GMT
recorded_with: VCR 6.0.0
