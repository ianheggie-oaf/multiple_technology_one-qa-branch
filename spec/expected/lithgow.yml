---
- council_reference: CC006/25
  address: 1108 Pipers Flat Road PORTLAND NSW 2847
  description: 'Two sheds: one is 84m2 with awnings, the other is 162m2 with additional
    awnings.'
  info_url: https://lithgowcc-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=CC006%2F25&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-12'
- council_reference: CC007/25
  address: 22 Henrietta Street WALLERAWANG NSW 2845
  description: Retaining Wall
  info_url: https://lithgowcc-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=CC007%2F25&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-11'
- council_reference: CC008/25
  address: 35 Ilford Street PORTLAND NSW 2847
  description: Shed
  info_url: https://lithgowcc-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=CC008%2F25&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-06'
- council_reference: CC009/25
  address: Clareville 641 Sodwalls Road TARANA NSW 2787
  description: Patio
  info_url: https://lithgowcc-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=CC009%2F25&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-11'
- council_reference: CC010/25
  address: Naringla 1804 Great Western Highway MOUNT LAMBIE NSW 2790
  description: Shed
  info_url: https://lithgowcc-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=CC010%2F25&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-11'
- council_reference: CC011/25
  address: 11 Finlay Avenue LITHGOW NSW 2790
  description: Shed
  info_url: https://lithgowcc-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=CC011%2F25&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-13'
- council_reference: DA008/25
  address: Clareville 641 Sodwalls Road TARANA NSW 2787
  description: Addition of Patio
  info_url: https://lithgowcc-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=DA008%2F25&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-11'
- council_reference: DA009/25
  address: 343-363 Willow Vale Road WALLERAWANG NSW 2845
  description: Subdivision 2 lots
  info_url: https://lithgowcc-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=DA009%2F25&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-11'
- council_reference: DA010/25
  address: 11 Finlay Avenue LITHGOW NSW 2790
  description: Shed
  info_url: https://lithgowcc-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=DA010%2F25&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-17'
- council_reference: S68012/25
  address: Genowlan Road GLEN ALICE NSW 2849
  description: Solid Fuel Heater
  info_url: https://lithgowcc-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=S68012%2F25&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-13'
- council_reference: SubCert001/25
  address: 277 Noola Road BOGEE NSW 2849
  description: Subdivision Certificate - Exempt - Torrens - 3 Lots into 2
  info_url: https://lithgowcc-web.t1cloud.com/T1PRDefault/WebApps/eProperty/P1/eTrack/eTrackApplicationDetails.aspx?ApplicationId=SubCert001%2F25&f=%24P1.ETR.APPDET.VIW&r=P1.WEBGUEST
  date_scraped: '2019-05-15'
  date_received: '2025-02-12'
